clean-modular-code/checks-conditionals/python-function-checks.md

```diff
@@ -28,7 +28,7 @@ This lesson covers several strategies for making your code more robust and easie
 
 1. Use [`try/except blocks`](#try-except) rather than simply allowing errors to occur.
 1. [Make checks Pythonic](#pythonic-checks)
-1. [Fail fast](fail-fast)
+1. [Fail fast](#fail-fast-strategy)
```

Comment:
```suggestion
1. [Fail fast](fail-fast)
```

@willingc because we have targets setup, we can keep the links short! This allows us to change the heading without the target links changing so the links always work!!